src/pedals/pedal_board.rs

```diff
@@ -34,15 +34,21 @@ type BoxedPedal = std::boxed::Box<
 pub struct PedalBoard {
     pedals: Vec<BoxedPedal>,
     board_id: i64,
+    channel: usize, // Channel this board will be used on
 }
 
 impl PedalBoard {
-    pub fn new() -> PedalBoard {
+    pub fn new(chan: usize) -> PedalBoard {
```

Comment:
I see modified tests that accept the new ctor param, but no tests that depend on the value of channel. Though I'm not sure that it matters if it just a set and read only value, but I wanted to have at least one comment :-)

Reply:
I see.  I just dropped that chan in so I could set it in the pedal board creation thread so the audio thread would know what channel to assign it to.  Then I could just send PedalBoards through the mpc::channel instead of making a new structure with the index and the PedalBoard.  Kind of lazy, but...

Reply:
I hear ya. If it's basically treated as an immutable ID field, and has a simple getter, it's nothing that needs test coverage. 

If you want to get pedantic, you could create some sort of factory that ensures no two PedalBoards get created with duplicate channels, since that would probably confuse downstream consumers. Then again, I see that the code today simply creates a hardcoded board on each of channels 0 and 1, so as long as that's the case, it doesn't really matter.